keyboards/ymd96/info.json

```diff
@@ -0,0 +1,16 @@
+        },
+        "LAYOUT_hgoel89": {
```

Comment:
This layout macro name `LAYOUT_hgoel89` does not match any of the existing ones either.